app/services/config_report_importer.rb

```diff
@@ -0,0 +1,41 @@
+class ConfigReportImporter < ReportImporter
+  attr_reader :report
+  def self.authorized_smart_proxy_features
+    ['Puppet']
+  end
+
+  def report_name_class
+    ConfigReport
+  end
+
+  def import
+    @report = super
```

Comment:
It is set, yet not available unless I set ```@report = super``` (so it fails on undefined persisted? for nil class... )
Maybe because it is not in the same context?